pkg/api/unversioned/generated.proto

```diff
@@ -232,6 +232,24 @@ message ListMeta {
   optional string resourceVersion = 2;
 }
 
+// MicroTime is version of Time with microsecond level precision.
+// 
+// +protobuf.options.marshal=false
+// +protobuf.as=Timestamp
+// +protobuf.options.(gogoproto.goproto_stringer)=false
+message MicroTime {
```

Comment:
For consistency with the rest of the system. I didn't want to spend time to check what I'd need to do to serialize our custom go/json to this proto.